.gitignore

```diff
@@ -1,5 +1,6 @@
 # Miscellaneous
 *.class
+*.lock
```

Comment:
Neet to comment out this line code to keep using same version of package in each developer